code/modules/mining/aux_base.dm

```diff
@@ -0,0 +1,338 @@
+///Mining Base////
+
+/area/shuttle/auxillary_base
+	name = "Auxillary Base"
+	luminosity = 0 //Lighting gets lost when it lands anyway
+	#define BAD_ZLEVEL	1
```

Comment:
Move these to the top of the file please, the indentation doesn't really mean anything and let's stick to conventions.